lib/puppet-lint/plugins/check_deprecated_functions/deprecated_functions.rb

```diff
@@ -0,0 +1,27 @@
+# Public: A puppet-lint custom check to detect deprecated functions.
+DEPRECATED_FUNCTIONS_VAR_TYPES = Set[:FUNCTION_NAME]
+
+# These functions have been deprecated in stdlib and will impact puppet upgrades
+# between puppet 7 and puppet 8.
+EASY_FUNCTIONS = [
+  'strip', 'rstrip', 'getvar', 'sort', 'upcase', 'round', 'chop', 'chomp',
+  'ceiling', 'capitalize', 'cammelcase', 'is_array', 'cod',
+  'min', 'max', 'lstrip', 'hash', 'has_key', 'downcase', 'abs', 'dig',
+  'dig44', 'unique'
+].freeze
```

Comment:
Could you please sort the list alphabetically?

```suggestion
EASY_FUNCTIONS = ['abs', 'camelcase', 'capitalize', 'ceiling', 'chomp', 'chop', 'cod', 'dig', 'dig44', 'downcase', 'getvar', 'has_key', 'hash', 'is_array', 'lstrip', 'max', 'min', 'round', 'rstrip', 'sort', 'strip', 'unique', 'upcase'].freeze
```